hack/windows/Install-OVS.ps1

```diff
@@ -251,6 +256,13 @@ function ConfigOVS() {
     ovs-vsctl --no-wait set Open_vSwitch . ovs_version=$OVS_VERSION
 }
 
+if ($InstallUserspace -eq $false) {
+    if(!(Get-Process containerd )){
```

Comment:
I don't think this check is requierd. Using containerd host process to run OVS userspace is one solution. But for this script, we should not assume such dependencies.